roles/pulibrary.pas/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Install grunt-cli
+  npm: name=grunt-cli global=yes
+  become: true
+
+- name: install sass
+  gem:
+    name: sass
+    state: latest
+    user_install: no
+
+# Didn't clone https://github.com/pulibrary/pas.git
```

Comment:
I couldn't get the clone to work. The repo is pretty large. It hung for a long time on https. When I tried an SSH style URL I got:

```fatal: [default]: FAILED! => {"failed": true, "msg": "to use the 'ssh' connection type with passwords, you must install the sshpass program"}```

Reply:
It is a private repo. But I would have expected https would have just given me the username/password dialog.

Reply:
will it always be private? in psuedo code
```
- name: add github ssh key
    copy: >
      src=files/id_rsa.github
      dest=/home/pulsys/.ssh/id_rsa.github
      owner=pulsys
      group=root
      mode=0600
  
  - name: configure ssh to use ansible key for github.com
    template: >
      src=templates/ssh_config.j2
      dest=/home/pulsys/.ssh/config
      owner=pulsys
      group=pulsys
      mode=0644
      
  - name: clone a private repository
    git: >
      repo=ssh://git@github.com/pulibrary/pas.git
      key_file=/home/pulsys/.ssh/id_rsa.github
      dest=/path/to/pas
```